prow/janitor.sh

```diff
@@ -88,13 +88,53 @@ for rule in $FIREWALL_RULES ; do
 done
 echo "Done cleaning up Firewall rules"
 
+### Forwarding Rules ###
+
+TARGET_POOLS=$(gcloud compute target-pools list \
+    --regions="${REGION}" \
+    --filter="instances:(gke-e2e-cloud-run) \
+    AND creationTimestamp < ${LIMIT_DATE}" \
+    --format='value(name)')
+
+for targetpool in $TARGET_POOLS; do
+  echo "Detected cloud run target pool ${targetpool}, querying forwarding rule"
+  forwardingitem=$(gcloud compute forwarding-rules list \
+    --filter=TARGET="https://www.googleapis.com/compute/v1/projects/$PROJECT/regions/$REGION/targetPools/$targetpool" \
+    --format='value(name)')
+  echo "Deleting forwarding rule ${forwardingitem}"
+  gcloud compute forwarding-rules delete "${forwardingitem}" \
+    --region="${REGION}" \
+    --quiet
+done
+echo "Done cleaning up forwarding rules"
+
+### Target Pools ###
+# Source: https://gist.github.com/prasvats/b2a4e33ad12b40191dd9d7e222d1abde
+
+TARGET_POOLS=$(gcloud compute target-pools list \
```

Comment:
These two sections of code are similar,  not asking to share them,  maybe add some comment.

Reply:
Done, clarified